app/src/main/java/dgca/verifier/app/android/verification/VaccinationViewHolder.kt

```diff
@@ -39,15 +39,15 @@ class VaccinationViewHolder(private val binding: ItemVaccinationBinding) :
 
     @SuppressLint("SetTextI18n")
```

Comment:
Will there be i18n?

Reply:
This it temporary. I'll move all string resources to `strings` file so we  can localize them later.

Reply:
Ok. Should at least be technically prepared to be easily done in concrete implementation.